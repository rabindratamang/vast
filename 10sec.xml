This XML file does not appear to have any style information associated with it. The document tree is shown below.
<VAST xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="3.0" xsi:noNamespaceSchemaLocation="vast.xsd">
	<Ad id="1" sequence="1">
		<InLine>
			<AdSystem>
				<![CDATA[ Reviveadservermod.com ]]>
			</AdSystem>
			<AdTitle>
				<![CDATA[ ad 2 ]]>
			</AdTitle>
			<Impression>
				<![CDATA[ https://krish.g-mana.net/media/debug/impression?impression=1&adId=1 ]]>
			</Impression>
			<Creatives>
				<Creative>
					<Linear>
						<Duration>
							00:00:10
						</Duration>
						<TrackingEvents>
							<Tracking event="start">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=start&adId=1 ]]>
							</Tracking>
							<Tracking event="firstQuartile">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=firstquartile&adId=1 ]]>
							</Tracking>
							<Tracking event="midpoint">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=midpoint&adId=1 ]]>
							</Tracking>
							<Tracking event="thirdQuartile">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=thirdquartile&adId=1 ]]>
							</Tracking>
							<Tracking event="complete">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=complete&adId=1 ]]>
							</Tracking>
							<Tracking event="pause">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=pause ]]>
							</Tracking>
							<Tracking event="mute">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=mute ]]>
							</Tracking>
							<Tracking event="fullscreen">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=fullscreen ]]>
							</Tracking>
							<Tracking event="unmute">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=unmute ]]>
							</Tracking>
							<Tracking event="closeLinear">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=closeLinear ]]>
							</Tracking>
							<Tracking event="acceptInvitationLinear">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=acceptInvitationLinear ]]>
							</Tracking>
							<Tracking event="skip">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=skip ]]>
							</Tracking>
							<Tracking event="progress">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=progress ]]>
							</Tracking>
							<Tracking event="exitFullscreen">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=exitfullscreen ]]>
							</Tracking>
							<Tracking event="rewind">
								<![CDATA[ https://krish.g-mana.et/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=rewind ]]>
							</Tracking>
							<Tracking event="resume">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=resume ]]>
							</Tracking>
						</TrackingEvents>
						<MediaFiles>
							<MediaFile delivery="progressive" bitrate="128000" width="1280" height="720" type="video/mp4" scalable="true" maintainAspectRatio="true">
								<![CDATA[ https://cdn.g-mana.online/media/assets/IZ%20juice%20it's%20simply%20amazing%20!%2010%20sec%20Ad.mp4 ]]>
							</MediaFile>
						</MediaFiles>
					</Linear>
				</Creative>
			</Creatives>
		</InLine>
	</Ad>
</VAST>
