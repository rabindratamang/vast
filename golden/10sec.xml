<VAST xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="3.0" xsi:noNamespaceSchemaLocation="vast.xsd">
	<Ad id="1012312454554" sequence="10123">
		<InLine>
			<AdSystem>
				<![CDATA[ Reviveadservermod.com ]]>
			</AdSystem>
			<AdTitle>
				<![CDATA[ golden 10 sec ]]>
			</AdTitle>
			<Impression>
				<![CDATA[ https://krish.g-mana.net/media/debug/impression?impression=1&adId=1 ]]>
			</Impression>
			<Creatives>
				<Creative id="golden10sec">
					<Linear>
						<Duration>
							00:00:10
						</Duration>
						<TrackingEvents>
							<Tracking event="start">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=start&adId=1 ]]>
							</Tracking>
							<Tracking event="firstQuartile">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=firstquartile&adId=1 ]]>
							</Tracking>
							<Tracking event="midpoint">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=midpoint&adId=1 ]]>
							</Tracking>
							<Tracking event="thirdQuartile">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=thirdquartile&adId=1 ]]>
							</Tracking>
							<Tracking event="complete">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?event=complete&adId=1 ]]>
							</Tracking>
							<Tracking event="pause">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=pause ]]>
							</Tracking>
							<Tracking event="mute">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=mute ]]>
							</Tracking>
							<Tracking event="fullscreen">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=fullscreen ]]>
							</Tracking>
							<Tracking event="unmute">
								<![CDATA[ https://jsonplaceholder.typicode.com/todos/1?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=unmute ]]>
							</Tracking>
							<Tracking event="closeLinear">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=closeLinear ]]>
							</Tracking>
							<Tracking event="acceptInvitationLinear">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=acceptInvitationLinear ]]>
							</Tracking>
							<Tracking event="skip">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=skip ]]>
							</Tracking>
							<Tracking event="progress">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=progress ]]>
							</Tracking>
							<Tracking event="exitFullscreen">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=exitfullscreen ]]>
							</Tracking>
							<Tracking event="rewind">
								<![CDATA[ https://krish.g-mana.et/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=rewind ]]>
							</Tracking>
							<Tracking event="resume">
								<![CDATA[ https://krish.g-mana.net/media/debug/impression/fc.php?script=rmvideoAds:rmvastEvent&bannerid=10&zoneid=4&event=resume ]]>
							</Tracking>
						</TrackingEvents>
						<MediaFiles>
							<MediaFile delivery="progressive" bitrate="128000" width="1280" height="720" type="video/mp4" scalable="true" maintainAspectRatio="true">
								<![CDATA[ https://raw.githubusercontent.com/rabindratamang/vast/master/assets/10.mp4 ]]>
							</MediaFile>
						</MediaFiles>
						<VideoClicks>
							<ClickThrough id="GDFP"><![CDATA[https://pubads.g.doubleclick.net/pcs/click?xai=AKAOjssQX4rH_q6lmmoN_CtZ3lyhvDty7xb_l45NQRIC1YEVRLjTmCoJDp5bBMKpvyMMthrdxNsl1CawdAiEw05vTWVElgnX6DSe6qgWHGtcr7iw3dSuK95H8KzRWNsYIVIA0DDaYz5RAbTZXeALPFdwafWLk8s6oBqiCh01e9eY1OoDWSE_AmkXrAVgF9ed-dc5N2ky2r--Al20FTxmbchmIwxiZVfLxPBS69r5eX9rzavPALpqZ029gLSkAeK9IhKPCb8EXgJSauRpJoQLvKBq9EJhTmWAVh-llAlwlkgQYZTxjfvXBlDaGVfWl8v2YDX9&sai=AMfl-YQzAWz_nkXZBYhIjrj5gkNo-UwvrMVJXgI9liQKJ2rVLm0ck1XyKVkR7kQbraGuJxUN6m78NUAY_2DgwH-MQImYoBgCvoPwnm7tHbhrjFod3u0_oA&sig=Cg0ArKJSzFb9j_UFoxBX&fbs_aeid=[gw_fbsaeid]&adurl=https://www.teva.co.il/about-teva/csr/wise-use-of-medicines/]]></ClickThrough>
							<ClickTracking id="GDFP"><![CDATA[https://ib.adnxs.com/getuid?https://api.permutive.com/v2.0/px/track?k=efd364e0-59cb-4b1c-98ac-37848277a8a4&i=$UID&e=VideoEvents&p=%257B%2522blockid%2522%253A11152%252C%2522videoevent%2522%253A%2522click%2522%257D&it=appnexus]]></ClickTracking>
						</VideoClicks>
					</Linear>
				</Creative>
			</Creatives>
		</InLine>
	</Ad>
</VAST>
